relay-sampling/src/condition.rs

```diff
@@ -282,8 +282,35 @@ impl RuleCondition {
 
 #[cfg(test)]
 mod tests {
+    use std::collections::BTreeMap;
+
+    use relay_base_schema::project::ProjectKey;
+    use uuid::Uuid;
+
+    use crate::dsc::TraceUserContext;
+    use crate::tests::{and, eq, glob, not, or};
+    use crate::DynamicSamplingContext;
```

Comment:
I was debating the idea of making a dummy Getter struct, just to make it more clear that this crate shouldn't care about implementations, but since we want to keep `DynamicSamplingContext`  in this crate then it's simpler just to use that. All the tests that used `Event` I've replaced with dsc.